cmd/tusd/cli/log.go

```diff
@@ -2,14 +2,14 @@ package cli
 
 import (
 	"log"
-	"os"
 
 	"github.com/tus/tusd"
 )
 
-var stdout = log.New(os.Stdout, "[tusd] ", 0)
-var stderr = log.New(os.Stderr, "[tusd] ", 0)
```

Comment:
Would you mind moving the definition of stdout and stderr back into log.go? I think the semantically belong into this file since they are used for logging. Furthermore, these values are also used in composer.go, so they don't belong to hooks.go

Reply:
Will do.